pitch.md

```diff
@@ -23,38 +79,101 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
 important to users and what kinds of values those properties might have._ 
 
+For now, and possibly always, I want to focus this language on beginners.
+With that in mind, I think the user will only be able to describe a few higher
+level LED actions like cascade, wave, fade, flash, and strip color. Various
+commands will require different properties but some would be color of the
+action (or range of colors for fade etc.), for how long the action should last,
+the brightness of the LEDs, and how wide the action should be - a 6 led wide 
+cascade or a single led etc.
```

Comment:
I think this is an excellent starting point. We want to start small and build up as needed. A thing I am curious about is whether or not you will also reward users experienced in making LED light shows. These kind of users are part of your domain, and with their knowledge, I wonder if you can design part of your DSL that rewards them as well. That way, they can make intricate light shows easily.